xcodecommand/xcbeautify.go

```diff
@@ -52,6 +52,7 @@ func (c *XcbeautifyRunner) Run(workDir string, xcodebuildArgs []string, xcbeauti
 		Stdin:  pipeReader,
 		Stdout: os.Stdout,
 		Stderr: os.Stderr,
+		Env:    xcodeCommandEnvs,
```

Comment:
Can we rename `xcodeCommandEnvs` to something like `commandEnvs` or maybe even `unbufferedIOEnv` because that is the single env it contains? Just asking because it is not only xcode now who will use the same env set.

Reply:
Renamed, it is more specific naming this way.